add doom template

MArpogaus opened the pull request:
This commit adds my base16 template for the doom emacs distribution.

MArpogaus commented:
Is there any Problem with this PR?

joshgoebel commented:
Please provide link to PR against the templates repo as well to show you've added these there.

belak commented:
Sorry for the delay, I just haven't gotten around to this yet.

Is there a reason you went with a new doom-specific theme over base16-emacs?

MArpogaus commented:
> Please provide link to PR against the templates repo as well to show you've added these there.

you can find it [here](https://github.com/chriskempson/base16-templates-source/pull/95)

MArpogaus commented:
> Sorry for the delay, I just haven't gotten around to this yet.
> 
> Is there a reason you went with a new doom-specific theme over base16-emacs? Also, with how your base16-terminal repo works, would it be better-off called base16-tty? I think "terminal" is generally a bit of an overloaded term when working with software, and when I think of it I generally think of different terminal emulators rather than a linux tty directly. What do you think?

Good point. I renamed the repo and updated the files accordingly.

MArpogaus commented:
Is there still a problem with my PRs?

belak commented:
Thanks for making those changes, I think it looks good now - I was just wondering why you made a new theme rather than contributing to base16-emacs.

There's a longer delay than normal because I've been debating what to do (chriskempson has been MIA for quite a while but it would be nice to add more collaborators). I haven't forgotten about this, it just hasn't been a super high priority yet.
